src/generic_ui/scripts/user.ts

```diff
@@ -41,46 +40,61 @@ module UI {
       }
       this.name = profile.name;
       this.imageData = profile.imageData || UI.DEFAULT_USER_IMG;
-      this.isOnline = profile.isOnline;
     }
 
     // Returns two strings, where each matches an array name in model.contacts.
     // Does not use an enum because we need a string value, and typescript
     // enums evaluate to numbers.
```

Comment:
I guess I've seen the numbers primarily when trying to understand the interaction between the app and extension in the browser console, where they aren't meaningful and hard to map back in my mind to the code I'm looking at.  Can't the enum have string values?  (e.g. Color.Red = "RED")
```javascript
var Color = {
  Red: 'RED',
  Blue: 'BLUE',
  ...
};
```